Assets/HoloToolkit/Utilities/Editor/ConfigureMenu.cs

```diff
@@ -187,5 +190,67 @@ private static void EnableVirtualReality()
                 Debug.LogException(e);
             }
         }
+
+        [MenuItem("HoloToolkit/Configure/Apply HoloLens Capability Settings", priority = 1)]
+        static void ApplyHoloLensCapabilitySettings()
+        {
+            AutoConfigureMenu window = (AutoConfigureMenu)EditorWindow.GetWindow(typeof(AutoConfigureMenu), true, "Apply HoloLens Capability Settings");
+            window.Show();
+        }
+
+        void OnGUI()
+        {
+            capabilityToggle(PlayerSettings.WSACapability.Microphone, @"Microphone
+
+Required for access to the HoloLens microphone.
+This includes behaviors like DictationRecognizer,
+GrammarRecognizer, and KeywordRecognizer.
+This capability is NOT required for the 'Select' keyword.
+
+Recommendation: Only enable if your application 
+needs access to the microphone beyond the
+'Select' keyword. The microphone is considered a 
+privacy sensitive resource.");
+
+            capabilityToggle(PlayerSettings.WSACapability.SpatialPerception, @"SpatialPerception
+
+Required for access to the HoloLens world mapping
+capabilities. These include behaviors like
+SurfaceObserver, SpatialMappingManager and 
+SpatialAnchor. 
+
+Recommendation: Enabled, unless your application
+doesn't use spatial mapping or spatial collisions
+in any way. ");
+
+            capabilityToggle(PlayerSettings.WSACapability.WebCam, @"WebCam
+
+Required for access to the HoloLens RGB camera 
+(also known as the locatable camera). This 
+includes APIs like PhotoCapture and VideoCapture.
+This capability is NOT required for mixed reality 
+streaming or for capturing photos or videos using
+the start menu. 
+
+Recommendation: Only enable if your application 
+needs to programmatically capture photos or 
+videos from the RGB camera. The RGB camera is
+considered a privacy sensitive resource.");
+
+            capabilityToggle(PlayerSettings.WSACapability.InternetClient, @"Internet Client
+
+Required if your application needs to access 
+the Internet. 
+
+Recommendation: Leave unchecked unless your 
```

Comment:
Internet Client is also needed for using a DictationRecognizer, which might be a less-obvious reason to check it. You'll also need it checked to use Unity's profiler. Up to you if these are worth adding to the description!